aptos-move/framework/aptos-token/sources/token_event_utils.move

```diff
@@ -0,0 +1,318 @@
+/// This module provide utils to create and emit token events
+module aptos_token::token_event_utils {
+    use std::string::String;
+    use std::signer;
+    use aptos_framework::event::{Self, EventHandle};
+    use aptos_framework::account;
+    use std::option::Option;
+    use aptos_std::any::Any;
+    use std::option;
+
+    friend aptos_token::token;
+
+    //
+    // Collection mutation events
+    //
+
+    /// Event emitted when collection description is mutated
+    struct CollectionDescriptionMutateEvent has drop, store {
```

Comment:
May be log the old description as it's no longer accessible on chain? Same for other events below